src/services/profiles/profiles.ts

```diff
@@ -2,14 +2,28 @@ import type { HttpInstance } from '../../types';
 import { rqlBuilder } from '../../rql';
 import { ProfilesService, Profile } from './types';
 import { HttpClient } from '../http-client';
-import { findAllIterator, findAllGeneric } from '../helpers';
+import {
+  findAllIterator,
+  findAllGeneric,
+  addCustomPropertiesToConfig,
+} from '../helpers';
 
 export default (
   client: HttpClient,
   httpAuth: HttpInstance
 ): ProfilesService => ({
   async find(options) {
-    return (await client.get(httpAuth, `/${options?.rql || ''}`, options)).data;
+    const customProperties = [
+      'data.custom_fields',
+      'data.groups.custom_fields',
```

Comment:
The `.*` is now implicit at the end of every string . `custom_fields` is still converted to `customFields`